specification/iothub/resource-manager/Microsoft.Devices/stable/2018-04-01/iothub.json

```diff
@@ -3118,6 +3118,32 @@
         "message": {
           "description": "Routing message",
           "$ref": "#/definitions/RoutingMessage"
+        },
+        "twin": {
+          "description": "Routing Twin Reference",
+          "$ref": "#/definitions/RoutingTwin"
+        }
+      }
+    },
+    "RoutingTwin": {
+      "description": "Twin reference input parameter. This is an optional parameter",
+      "type": "object",
+      "properties": {
+        "tags": {
+          "description": "Twin Tags",
+          "type": "object"
```

Comment:
>"type": "object" [](start = 10, length = 16)

why is everything of type object?

Reply:
Twin is a IoTHub primitive that takes form of three properties which can be any valid JSON object. Thats why we have defined in swagger as objects.